NtupleTools/python/parameters/zz.py

```diff
@@ -0,0 +1,63 @@
+# Parameters to be used in production of ZZ ntuples
+# Only parameters seen here are used. make_ntuples_cfg.py loads these first
+# and then loads any modifications to these parameters from a custom param file
+# passed via paramFile=/path/to/param/file.py
+
+from FinalStateAnalysis.Utilities.cfgtools import PSet
+from collections import OrderedDict
+
+parameters = {
+    # selections on all objects whether they're included in final states or not, done immediately after necessary variables are embedded
+    'preselection' : OrderedDict(
+        [
+            # Veto electrons that are very close to muons
+            ('e', {
+                    'm' : {
+                        'deltaR' : 0.05,
+                        'selection' : 'userFloat("HZZ4lIDPassTight") > 0.5',
+                        },
+                    },
+             ),
+            # Remove jets that are near tight ID'd, no-FSR isolated leptons
+            ('j', {
+                    'selection' : 'pt > 30 && eta < 4.7 && eta > -4.7 && userFloat("puID") > 0.5',
```

Comment:
Perhaps here on the jet issue. There is no corresponding jet selection in the default. How is this selection used?